opencog/cython/PythonEval.cc

```diff
@@ -415,8 +414,7 @@ Handle PythonEval::apply(const std::string& func, Handle varargs)
     // Py_DECREF(pyDict);
 
     // If we can't find that function then throw an exception.
-    if (!pyUserFunc)
-    {
+    if (!pyUserFunc) {
```

Comment:
coding style regression? after all that talk about style and readability and cognitive load, why would you do this?